### DISCUSSION TITLE
AC-PC alignment not applied

### DISCUSSION THREAD

**Discussion by HdeV999** [brainvisa/brainvisa.github.io#129] [Q&A] [answered]
Hi, 
I'm having trouble with the AC-PC alignment. I've tried all three methods (manual, SPM and aimsMRregister) on a given subject. In each case, the detection of AC and PC seems reasonable, but the output image is not aligned with the AC-PC. The processes seem to run fine (including the "Applying transformation" step which takes ~5sec), but my output images and graphs are not rotated to align the AC with the PC. What am I missing? 
Thank you kindly

**Comment by denisri** [comment]
The "alignment" step in Morphologist does not aim to actually realign images, but to have an affine transformation which will allow to go to a common inter-subject coordinates system, and use it when needed (it is used internally in some segmentation steps and in sulci identification). Normally the outputs of Morphologist are all in the subject acquisition native space, but Braivisa provides a transformation to a Talairach or MNI space, which can be applied. Anatomist views for instance can be set in such a common referential (the red (our Talairach) or light blue (MNI) ones) and will then display in this chosen coordinates system, applying resamplings to images etc.

**Reply by HdeV999** [reply]
Thanks, that clarifies it for me. I still can't manage to obtain the visualization that I am looking for: after opening a hemisphere, I would like my cross-mesh cursor to be aligned with the AC-PC line, so that I can visualize the VAC line (vertical line starting from the AC and perpendicular to the AC-PC line). Is there an easy way to get either the brain or the cursor rotated to visualize this VAC line? 
(When loading the Talairach-AC-PC referential the cursor is still not aligned with the AC-PC.) (By the way, I am trying to solve this for someone using a non-developer BrainVISA installation)

**Comment by denisri** [comment, marked as answer]
There are 2 use cases. Or 3 actually:
1. you are in brainvisa, and you display the data (image, mesh, sulci graph, whatever) using the eye icon (a brainvisa viewer process): then the data should be assigned a non-default referential (the colored ball next to the data object in anatomist should **not** be red), then click on the referentialbutton of the window (the colored bar between the menu and the 3D view) and select either the red (Talairach AC/PC) or light blue (MNI) referential): the view should reorient the data and the cursor should be aligned to the axis you want.
2. You are not in brainvisa, and have loaded the data in Anatomist "by hand". Then the data will be in the default (red) referential. Select the data, right click and select "referential/apply information from header" in the popup menu. If the information is actually contained in the data (which should be the case in data coming from Morphologist), the data will get another referential (another color) with transformation to Talairach AC/PC and MNI referentials, and you can get to case 1.
3. If you are not in brainvisa, and the data loaded in Anatomist doesn't have the transformation in its header, you can create a new referential for the data, open the referentials/transformations view (in the settings menu of the main window of Anatomist), then load the transfomation (.trm file saved in morphologist in `subject/acquisition/registration/` directory) between the data ref and the Talairach or MNI ref by dragging an arron in the referentials window.

**Reply by HdeV999** [reply]
Thanks! I was trying the first option but was not loading the referentials correctly. Now it works wonders!
